multimodal-understanding/repeatable-patterns/10-multimodal-rag-with-langchain/multimodal_rag_with_nova.ipynb

```diff
@@ -0,0 +1,767 @@
+{
```

Comment:
<p>why they are commented out here? it might make sense to have a requirements.txt in the directory to lock the versions since langchain is updating/deprecating quite frequently.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws-samples/amazon-nova-samples/pull/18/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>